### ISSUE TITLE
The project nav isn't misaligned 

### ISSUE THREAD

**Issue by heathermcd12** [heathermcd12/portfolio#16]
http://heatherlynn.ca/work/begin-again/

<img width="1435" alt="screen shot 2016-03-25 at 4 16 49 pm" src="https://cloud.githubusercontent.com/assets/6333582/14053038/0a1a7c6c-f2a5-11e5-8c56-fff9d068dd55.png">

The "previous" - "next" links are misaligned - But look perfect when Im in my terminal mode?? 

<img width="1436" alt="screen shot 2016-03-25 at 4 16 55 pm" src="https://cloud.githubusercontent.com/assets/6333582/14053039/0fdfb0b8-f2a5-11e5-9d2c-20c2540ef0ba.png">

@thomasjbradley

**Comment by thomasjbradley** [comment]
So they're supposed to be far apart?
If so, then they're working for me—I expect it was a browser caching thing.

**Comment by heathermcd12** [comment]
Perfect - It seems that github is having a hard time making all my new commits and code work for a day or too. Its happening again with an image/link. 

Here is it before I committed my code:

<img width="783" alt="screen shot 2016-03-27 at 9 28 03 am" src="https://cloud.githubusercontent.com/assets/6333582/14065591/49e85026-f3fe-11e5-93e1-6d5d4fcebc4c.png">

Here it is in heatherlynn.ca, 12 or so hours after I committed it:

<img width="767" alt="screen shot 2016-03-27 at 9 28 16 am" src="https://cloud.githubusercontent.com/assets/6333582/14065596/6d8afde4-f3fe-11e5-95a5-19f944036efa.png">

**Comment by heathermcd12** [comment]
Fixed!

**Comment by thomasjbradley** [comment]
Yep, it's supposed to do that—it's making your website load significantly faster.
If you want to see the changes immediately go into CloudFlare and empty the caches.
